src/main/java/com/bytecoders/pharmaid/repository/model/User.java

```diff
@@ -28,4 +28,7 @@ public class User {
   @Column(name = "hashed_password", nullable = false)
   @JsonIgnore
   private String hashedPassword;
+
+  @Column(name = "user_type", nullable = false)
+  private Integer userType = 0;
```

Comment:
FYI the db was throwing an error due to adding a non-nullable field to `users` table.

<img width="1423" alt="image" src="https://github.com/user-attachments/assets/cb091276-b826-494a-9f6a-c78b4cb1e962">


Fixed with:
```
ALTER TABLE users ADD COLUMN user_type integer NOT NULL DEFAULT 0;

ALTER TABLE users ALTER COLUMN user_type DROP DEFAULT;
```